src/solver/conv_hip_implicit_gemm_v4r4_gen_xdlops_wrw_fp32.cpp

```diff
@@ -0,0 +1,604 @@
+        std::string(" -DCK_PARAM_PROBLEM_N=") + std::to_string(n) +
+        std::string(" -DCK_PARAM_PROBLEM_C=") + std::to_string(c) +
+        std::string(" -DCK_PARAM_PROBLEM_K=") + std::to_string(k) +
+        std::string(" -DCK_PARAM_PROBLEM_Y=") + std::to_string(y) +
+        std::string(" -DCK_PARAM_PROBLEM_X=") + std::to_string(x) +
+        std::string(" -DCK_PARAM_PROBLEM_HI=") + std::to_string(hi) +
+        std::string(" -DCK_PARAM_PROBLEM_WI=") + std::to_string(wi) +
+        std::string(" -DCK_PARAM_PROBLEM_HO=") + std::to_string(ho) +
+        std::string(" -DCK_PARAM_PROBLEM_WO=") + std::to_string(wo) +
+        std::string(" -DCK_PARAM_PROBLEM_CONV_STRIDE_H=") + std::to_string(conv_stride_h) +
+        std::string(" -DCK_PARAM_PROBLEM_CONV_STRIDE_W=") + std::to_string(conv_stride_w) +
+        std::string(" -DCK_PARAM_PROBLEM_CONV_DILATION_H=") + std::to_string(conv_dilation_h) +
+        std::string(" -DCK_PARAM_PROBLEM_CONV_DILATION_W=") + std::to_string(conv_dilation_w) +
+        std::string(" -DCK_PARAM_PROBLEM_LEFT_PAD_H=") + std::to_string(in_left_pad_h) +
+        std::string(" -DCK_PARAM_PROBLEM_LEFT_PAD_W=") + std::to_string(in_left_pad_w) +
+        std::string(" -DCK_PARAM_PROBLEM_RIGHT_PAD_H=") + std::to_string(in_right_pad_h) +
+        std::string(" -DCK_PARAM_PROBLEM_RIGHT_PAD_W=") + std::to_string(in_right_pad_w) +
+        std::string(" -DCK_PARAM_PROBLEM_CONV_GROUP_COUNTS=") + std::to_string(ctx.group_counts) +
+        std::string(" -DCK_PARAM_TUNABLE_BLOCK_SIZE=") + std::to_string(block_size) +
+        std::string(" -DCK_PARAM_TUNABLE_GEMM_M_PER_BLOCK=") + std::to_string(config.GemmMPerBlock) +
+        std::string(" -DCK_PARAM_TUNABLE_GEMM_N_PER_BLOCK=") + std::to_string(config.GemmNPerBlock) +
+        std::string(" -DCK_PARAM_TUNABLE_GEMM_K_PER_BLOCK=") + std::to_string(config.GemmKPerBlock) +
+        std::string(" -DCK_PARAM_TUNABLE_GEMM_K_BLOCKS=") + std::to_string(config.GemmKBlocks) +
+        std::string(" -DCK_PARAM_DEPENDENT_GRID_SIZE=") + std::to_string(grid_size) +
+        std::string(" -DCK_PARAM_GEMM_M_PER_WAVE=") + std::to_string(config.GemmMPerWave) +
+        std::string(" -DCK_PARAM_GEMM_N_PER_WAVE=") + std::to_string(config.GemmNPerWave) +
+        std::string(" -DCK_PARAM_TUNABLE_GEMM_B_BLOCK_COPY_CLUSTER_LENGTHS_GEMM_K=") + std::to_string(GemmBBlockCopyClusterLengths_GemmK) +
+        std::string(" -DCK_PARAM_TUNABLE_GEMM_B_BLOCK_COPY_CLUSTER_LENGTHS_GEMM_N=") + std::to_string(GemmBBlockCopyClusterLengths_GemmN) +
+        std::string(" -DCK_PARAM_TUNABLE_GEMM_A_BLOCK_COPY_CLUSTER_LENGTHS_GEMM_K=") + std::to_string(GemmABlockCopyClusterLengths_GemmK) +
+        std::string(" -DCK_PARAM_TUNABLE_GEMM_A_BLOCK_COPY_CLUSTER_LENGTHS_GEMM_M=") + std::to_string(GemmABlockCopyClusterLengths_GemmM) +
+        std::string(" -DCK_PARAM_TUNABLE_GEMM_B_BLOCK_COPY_SRC_DATA_PER_READ_GEMM=") + std::to_string(GemmBBlockCopySrcDataPerRead_GemmK) +
```

Comment:
As the loading dimension is not fixed in the future, we intend to remove the suffix.

Reply:
are you saying that we are going to make it tunable? If yes, I agree with your reasoning. Otherwise, it is more readable to put the dimension name like GemmK as suffix

Reply:
Yes. We need to make it tunable in the future.